packages/button/src/Button.js

```diff
@@ -42,83 +43,71 @@ type Props = {
   /** Display a primary button */
   primary?: boolean,
   /** Available sizes */
-  size?: 'small' | 'medium' | 'large',
+  size?: 'small' | 'medium' | 'large' | 'xlarge',
   /** Available types */
   type?: 'button' | 'submit',
   /** Available variants */
   variant?: 'regular' | 'danger' | 'success' | 'warning'
 };
 
 const buttonTheme = (props, baseTheme) => ({
-  Button_backgroundColor: baseTheme.color_gray_10,
-  Button_backgroundColor_active: baseTheme.color_theme_10,
-  Button_backgroundColor_focus: baseTheme.color_gray_10,
-  Button_backgroundColor_hover: baseTheme.color_gray_20,
-  Button_backgroundColor_minimal_active: baseTheme.color_theme_10,
-  Button_backgroundColor_minimal_hover: baseTheme.color_gray_20,
-  Button_backgroundColor_primary: baseTheme.color_theme_50,
-  Button_backgroundColor_primary_active: baseTheme.color_theme_30,
-  Button_backgroundColor_primary_focus: baseTheme.color_theme_50,
-  Button_backgroundColor_primary_hover: baseTheme.color_theme_40,
-  Button_backgroundImage_primary_active: 'none',
-  Button_backgroundImage_primary_hover: `radial-gradient(circle, ${baseTheme.color_theme_40} 0%, ${baseTheme.color_theme_50} 100%)`,
+  Button_backgroundColor: baseTheme.color_gray_20,
+  Button_backgroundColor_active: baseTheme.color_gray_30,
+  Button_backgroundColor_focus: baseTheme.color_gray_20,
+  Button_backgroundColor_hover: baseTheme.color_gray_10,
+  Button_backgroundColor_minimal_active: baseTheme.color_gray_20,
+  Button_backgroundColor_minimal_hover: baseTheme.color_gray_10,
+  Button_backgroundColor_primary: baseTheme.color_theme_70,
+  Button_backgroundColor_primary_active: baseTheme.color_theme_80,
+  Button_backgroundColor_primary_focus: baseTheme.color_theme_70,
+  Button_backgroundColor_primary_hover: baseTheme.color_theme_60,
   Button_borderColor: baseTheme.borderColor,
-  Button_borderColor_active: baseTheme.borderColor_active,
-  Button_borderColor_focus: baseTheme.borderColor_focus,
-  Button_borderColor_hover: baseTheme.borderColor,
-  Button_borderColor_primary: 'transparent',
-  Button_borderColor_primary_active: 'transparent',
-  Button_borderColor_primary_hover: 'transparent',
-  Button_borderColor_primary_focus: baseTheme.color_theme_100,
+  Button_borderColor_focus: baseTheme.color_white,
   Button_borderRadius: baseTheme.borderRadius_1,
   Button_borderWidth: '1px',
-  Button_boxShadow: baseTheme.shadow_1,
   Button_boxShadow_focus: `0 0 0 1px ${baseTheme.borderColor_focus}`,
-  Button_boxShadow_minimal_focus: `0 0 0 1px ${baseTheme.borderColor_focus}`,
-  Button_boxShadow_primary_focus: `0 0 0 1px ${baseTheme.color_theme_100}, rgba(0,0,0,0.25) 0 2px 2px`,
-  Button_color_text: baseTheme.color_gray_80,
+  Button_color_text: baseTheme.color_gray_100,
   Button_color_text_minimal: baseTheme.color_link,
   Button_color_text_primary: baseTheme.color_text_onprimary,
   Button_fontSize: baseTheme.fontSize_ui,
```

Comment:
`Button_fontSize` is only applied to Content, so should it be `ButtonContent_fontSize`?